packages/beacon-node/src/network/network.ts

```diff
@@ -411,6 +412,17 @@ export class Network implements INetwork {
           }
         }
       }
+
+      // If we are subscribed and post capella fork epoch, try gossiping the cached bls changes
+      if (
+        this.isSubscribedToGossipCoreTopics() &&
+        epoch >= this.config.CAPELLA_FORK_EPOCH &&
+        !this.cachedBlsChangesPromise
+      ) {
+        this.cachedBlsChangesPromise = this.gossipCachedBlsChanges().then(() => {
+          this.cachedBlsChangesPromise = null;
+        });
+      }
```

Comment:
Just move this code above under the `// On fork transition` comment, with some retry in case there are no peers